lib/coinbase/wallet.rb

```diff
@@ -363,9 +364,11 @@ def export
       Data.new(wallet_id: id, seed: @master.seed_hex)
     end
 
-    def faucet
+    def faucet(asset_id: '')
```

Comment:
Can we update the `address` faucet method and make this a delegator like all of the other default address method?

Reply:
Also this can be

```
def faucet(asset_id: nil)
  opts = { asset_id: asset_id }.compact
end
```